specification/ai/Face/tspconfig.yaml

```diff
@@ -29,7 +29,6 @@ options:
   "@azure-tools/typespec-ts":
     package-dir: "ai-vision-face-rest"
     generate-metadata: true
-    generate-test: true
```

Comment:
(same but tests) 
what does removing the property does? does the emitter by default generates the tests?

Reply:
yes, exactly. emitters handles the default with more properly
- `generate-test` would be `true` by default if emitter detects there is no test folder; if already having test folders this would be false.